regex-assembly/934101.ra

```diff
@@ -1,12 +1,16 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-close\s*\(
-exists\s*\(
-fork\s*\(
-open\s*\(
-read\s*\(
-spawn\s*\(
-watch\s*\(
-write\s*\(
-require\s*\(
+##!$ \s*\(
+
+##!> assemble
```

Comment:
No need to use an extra assemble block here.

Reply:
Removed!